pkgs/development/interpreters/python/cpython/2.7/default.nix

```diff
@@ -356,5 +356,9 @@ in with passthru; stdenv.mkDerivation ({
       # Higher priority than Python 3.x so that `/bin/python` points to `/bin/python2`
       # in case both 2 and 3 are installed.
       priority = -100;
+      knownVulnerabilities = [
+        "CVE-2022-37454"
```

Comment:
See https://github.com/NixOS/nixpkgs/pull/201859#issuecomment-1352180307, CVE-2022-37454 doesn't apply to Python 2.7.